### DISCUSSION TITLE
QOTO QT-05M lost connection

### DISCUSSION THREAD

**Discussion by Belaba-hub** [Koenkk/zigbee2mqtt#23935] [Devices]
Hi everyone,
I got a QOTO QT-05M (https://www.zigbee2mqtt.io/devices/QT-05M.html) and it joins my zigbee network perfectly but after a few days it losts the connections, but only in the receiving way. It means that if I open or close the valve with the button on the side of the QT-05M, then it sends it's state into the zigbee network, but if I try to control the valve from home assistant or from the GUI of the zigbee2mqtt, the command shows up in the zigbee2mqtt logs, but nothing happens on the QT-05M. It looks like it does not listening anymore over the zigbee network, only sends in it's status when it's necessary. If I rejoin the device to the zigbee network, it start working again. It looks like sometime the "disconnection" happens sooner if the device is connected to for example a IKEA TRADFI signal repeater or any other repeater, and later if it's connected directly to the Sonoff ZigBee 3.0 USB Dongle-E coordinator.
Is anyone else facing similar issue related to this device?

**Comment by Belaba-hub** [comment]
Anyone else with QOTO QT-05M?

**Comment by s482dcaw** [comment]
I have the same problem with this device. Happened suddenly, and I don't know how to fix it.
Any suggestions, welcome

Here are some logs:

38z2m: Publish 'set' 'valve_state' to 'Porch Water valve' failed: 'Error: ZCL command 0xa4c1385e49460c94/1 manuSpecificTuya.dataRequest({"seq":0,"dpValues":[{"dp":102,"datatype":2,"data":[0,0,0,5]}]}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"reservedBits":0,"writeUndiv":false}) failed (Data request failed with error: 'No network route' (205))'
error 2024-10-20 20:48:54z2m: Publish 'set' 'valve_state' to 'Porch Water valve' failed: 'Error: ZCL command 0xa4c1385e49460c94/1 manuSpecificTuya.dataRequest({"seq":1,"dpValues":[{"dp":102,"datatype":2,"data":[0,0,0,50]}]}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"reservedBits":0,"writeUndiv":false}) failed (Data request failed with error: 'No network route' (205))'
error 2024-10-20 20:49:09z2m: Publish 'set' 'valve_state' to 'Porch Water valve' failed: 'Error: ZCL command 0xa4c1385e49460c94/1 manuSpecificTuya.dataRequest({"seq":2,"dpValues":[{"dp":102,"datatype":2,"data":[0,0,0,0]}]}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"reservedBits":0,"writeUndiv":false}) failed (Data request failed with error: 'No network route' (205))'

One might think there is no network route, but it's not the case, because once I interact with the device manually and its status changes, the updates are coming. 

 2024-10-20 20:50:22z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/Porch Water valve', payload '{"battery":100,"last_watering_duration":141,"linkquality":91,"remaining_watering_time":0,"shutdown_timer":90,"valve_state":0,"valve_state_auto_shutdown":0,"water_flow":0}'
info 2024-10-20 20:50:23z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/Porch Water valve', payload '{"battery":100,"last_watering_duration":141,"linkquality":80,"remaining_watering_time":0,"shutdown_timer":90,"valve_state":0,"valve_state_auto_shutdown":0,"water_flow":0}'

**Reply by Belaba-hub** [reply]
Since aprox. a month I moved the QT-05M and now it's connected directly to the coordinator. Since the movement I opened and closed the valve many times - sometimes more than a week after the previous change - and it still works. Maybe the problem only happens when it's connected to a router in the network? 

Is yours connected directly to the coordinator?

**Reply by Belaba-hub** [reply]
One more thing: the workaround for me was to remove the device from the zigbee network and rejoin it again. After this, it worked again for a while, and stopped again.
